assets/src/edit-story/components/library/panes/shapes/shapePreview.js

```diff
@@ -30,37 +30,51 @@ import { PAGE_WIDTH } from '../../../../constants';
 import createSolidFromString from '../../../../utils/createSolidFromString';
 import LibraryMoveable from '../shared/libraryMoveable';
 import { useUnits } from '../../../../units';
+import { themeHelpers } from '../../../../../design-system';
 
 // By default, the element should be 33% of the page.
 const DEFAULT_ELEMENT_WIDTH = PAGE_WIDTH / 3;
 const PREVIEW_SIZE = 36;
 
-const createGrid = ({ columns, gap, minWidth }) => css`
+const createGrid = ({ columns, columnGap, rowGap, minWidth }) => css`
```

Comment:
💭 Should we add defaults here? I don't know if we need them but maybe it'd help?